src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx

```diff
@@ -32,9 +33,11 @@ export const ChangedAnnotationListItem = (props: {
   let annotationColumnHeader = undefined;
   let useOneLineRowClass = true;
   let defaultTitle = '';
+  let geneColumnIndex = 0;
```

Comment:
I like what you had previously on finding the gene index by the header name. Why changed to this implementation?

Reply:
reverted back. make it into an util func that takes in `annotationHeader` to output the index.